content/docs/migration_1.0/Concepts.md

```diff
@@ -0,0 +1,126 @@
+Zenoh does not impose any encoding requirement on the user, nor does it operate on it.  
+It can be thought of as optional metadata, carried over by Zenoh in such a way that the end user’s application may perform different operations based on encoding.  
+We have expanded our list of predefined encoding types from Zenoh 0.11.0 to include variants for numerous IANA standard encodings, including but not limited to  `video/x` , `application/x`, `text/x`, `image/x` and `audio/x` encoding families, as well as an encoding family specific to Zenoh defined by the prefix `zenoh/x` .   
+Users can also define their own encoding scheme that does not need to be based on the predefined IANA variants. Upon declaring a specific encoding, the users encoding scheme will be prefixed with `zenoh/bytes` for distinction.
+
+
+## Attachment
+
+We have made attachment more flexible across API’s, essentially accepting anything that can be converted to a `ZBytes` as an optional extra to `put` , `delete` , on `Query`'s and Query `reply` s.  
+We also allow for composite types to be converted into `ZBytes`, meaning that using the Attachment API as metadata is more easy than ever.
+
+## Query & Queryable
+
+`Query` and `Queryable` have been slightly reworked.  
+For the API replying to a `Query` from a `Queryable` declared on a session: 
+The `reply` function has been split into 3 separate functions variants depending on the type of reply the user wants to send.  
+`reply` , `reply_del`, `reply_err`  
+
+We have added the ability to get the underlying `Handler` of a Queryable as well.
+
+## Use accessors to get private members
+Across language bindings we encapsulate members of structs, and they can’t be accessed directly now.  
+The only way to access struct values is to use the getter function associated with them.
+
+
+## Pull Subscribers have been removed
+
+The concept of a pull subscriber no longer exists in Zenoh,
+However, when creating a `Subscriber`, it may be the case that developers only care about the latest data and want to discard the oldest data. 
+We can use `RingChannel` to get this behaviour.
+This contrasts a `FIFOChannel` which is the default channel type used internally in Subscribers
+You can take a look at examples of usage in any language’s examples/z_pull.x
+
+## Timestamps
+
+We now tie generating a timestamp to a Zenoh session, with the timestamp inheriting the `ZenohID` of the session.
+
+This will affect user-created plugins and applications that need to generate timestamps in their Storage and sending of data.  
+⚠️ Note: Timestamps are important for Storage Alignment and Replication. Data stored in Data bases must include a Timestamp to be properly aligned across Data Stores by Zenoh. 
+The `timestamping` configuration option must also be enabled for this.
+
+## Storages
+The storage-manager will fail to launch if the timestamping configuration option is disabled.  
+From Zenoh 1.0.0 user-applications can load plugins.  
```

Comment:
Let's have a different section for plugins.

Reply:
Sure thing. What else should we add to it ?

Reply:
No, you misunderstood my point. I meant to add section on how to load plugins from the API but not to cover actual plugin implementations.